frontend/src/types/pipeline/task.ts

```diff
@@ -65,6 +83,8 @@ export type TaskPayload =
   | TaskGeneralPayload
   | TaskDatabaseCreatePayload
   | TaskDatabaseSchemaUpdatePayload
+  | TaskDatabaseSchemaUpdateGhostPayload
+  | TaskDatabaseSchemaUpdateGhostCutoverPayload
```

Comment:
`TaskDatabaseSchemaUpdateGhostPayload` should be `TaskDatabaseSchemaUpdateGhostSyncPayload`.

There should be also a type named `TaskDatabaseSchemaUpdateGhostDropOriginalTablePayload`.